core/domain/activity_jobs_one_off.py

```diff
@@ -58,9 +58,7 @@ def reduce(key, values):
 
 
 class SnapshotMetadataModelsIndexesJob(jobs.BaseMapReduceOneOffJobManager):
-    """Job that indexes all explorations and collections and compute their
-    ranks.
-    """
+    """Job that reindexes all children of BaseSnapshotMetadataModel."""
```

Comment:
This job doesn't index anything -- it re-commits it. So:

Job that re-puts all children of BaseSnapshotMetadataModel (in order to, e.g., ensure that indexed fields are properly indexed).

Reply:
Done.